Tests/Unit/MSFT_xIISFeatureDelegation.tests.ps1

```diff
@@ -4,14 +4,30 @@ $global:DSCResourceName = 'MSFT_xIISFeatureDelegation'
 #region HEADER
 
 [String] $moduleRoot = Split-Path -Parent (Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path))
- if ( (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DSCResource.Tests'))) -or `
-      (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DSCResource.Tests\TestHelper.psm1'))) )
-{
-    & git @('clone','https://github.com/PowerShell/DscResource.Tests.git',(Join-Path -Path $moduleRoot -ChildPath '\DSCResource.Tests\'))
+$repoSource = (Get-Module -Name $global:DSCModuleName -ListAvailable)
+
+if ($repoSource.RepositorySourceLocation.Host -eq 'www.powershellgallery.com') {
```

Comment:
It would fail in an ugly way when indexing array's first element though, I think better way to write it would be:
```PowerShell
if (($repoSource -ne $null) -and ($repoSource[0].RepositorySourceLocation.Host -eq 'www.powershellgallery.com')) { ```